x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phase_footer/phase_footer.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { EuiIcon, EuiText, EuiLink, EuiPanel } from '@elastic/eui';
+
+import { PhasesExceptDelete } from '../../../../../../common/types';
+
+import { usePhaseTimings } from '../../form';
+
+import { InfinityIcon } from '../infinity_icon';
+import './phase_footer.scss';
+
+interface Props {
+  phase: PhasesExceptDelete;
+}
+
+export const PhaseFooter: FunctionComponent<Props> = ({ phase }) => {
+  const phaseTimings = usePhaseTimings();
+  const phaseConfiguration = phaseTimings[phase];
+  const setValue = phaseTimings.setDeletePhaseEnabled;
+
+  if (phaseConfiguration === 'disabled' || phaseConfiguration === 'enabled') {
+    return (
+      <EuiPanel className={'ilmPhaseFooter'} hasShadow={false}>
```

Comment:
nit; here and elsewhere. the convention for fixed string-literal props is:

```ts
className="ilmPhaseFooter" // without curly braces, with double quotes
```

I think this is perhaps webstorm specific behaviour as I recall running into it there. If you're using webstorm I managed to disable auto-bracket insertion for JSX by following this issue: https://youtrack.jetbrains.com/issue/WEB-31288?_ga=2.14889842.555971730.1612525382-1881428346.1584373905